python/surf/protocols/ssi/_SsiPrbsRateGen.py

```diff
@@ -52,6 +77,28 @@ def __init__(self, **kwargs):
             mode         = "RW",
         ))
 
+        def get_conv(var):
+            return clock_freq / (self.RawPeriod.value()+1)
```

Comment:
Do you need to save the clock_freq value as self.clock_freq? Just asking incase the context is lost after the __init__()